packages/next/generators.json

```diff
@@ -17,16 +17,6 @@
             "factory": "./src/generators/next-auth/generator",
             "schema": "./src/generators/next-auth/schema.json",
             "description": "Add NextAuth to a project"
-        },
-        "next-auth-redis": {
-            "factory": "./src/generators/next-auth-redis/generator",
-            "schema": "./src/generators/next-auth-redis/schema.json",
-            "description": "Add Redis for session management to NextAuth app"
-        },
-        "next-auth-redis-deployment": {
-            "factory": "./src/generators/next-auth-redis-deployment/generator",
-            "schema": "./src/generators/next-auth-redis-deployment/schema.json",
-            "description": "Add Redis infra to NextAuth app"
```

Comment:
Would also need to review the redis e2e tests, maybe commenting those out for now?